cfgov/jinja2/v1/_lib/django_calendar_event_processor.py

```diff
@@ -1,46 +0,0 @@
-import sys
```

Comment:
Yeah, don't think they're necessary. Thanks for pointing out the sheer_indexing issue though - fixing that now!